File: 준영/프로그래머스/LV1/최대공약수와 최소공배수.js
function gcd(x, y) {
  var temp
  while ((x % y) > 0) {
    temp = x % y
    x = y
    y = temp
  }
  return y
}

function solution(n, m) {
  var gcd_temp = gcd(n, m)
  var lcm_temp = parseInt((n * m) / gcd_temp)
  var answer = [gcd_temp, lcm_temp]
  return answer
}

File: 준영/프로그래머스/LV1/최대공약수와 최소공배수.py
def gcd(x, y):
    while y:
        x, y = y, x % y
    return x

def solution(n, m):
    gcd_temp = gcd(n, m)
    lcm_temp = (n * m) // gcd_temp
    answer = [gcd_temp, lcm_temp]
    return answer
